sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -469,6 +479,7 @@ export class AssessmentController {
    */
   private createOutputForProgramYearTotals(
     programYearTotalAwards: AwardTotal[],
+    ftProgramYearContributionTotal: FTProgramYearContributionTotal[],
```

Comment:
This method can receive only the `programYearTotals` which will contain the `contributionTotals` and `awardTotals`.